ci/deploy.sh

```diff
@@ -7,9 +7,16 @@ set -o errexit \
     -o nounset \
     -o pipefail
 
+# set environment variables for either Travis or GitHub Actions
+REPO_SLUG=${TRAVIS_REPO_SLUG:-$GITHUB_REPOSITORY}
```

Comment:
I assume the `:-` is like an `or`?

I agree with what you mentioned that this isn't super clean. One other option, I think, is to just leave the github ones, and make comments about how to fill them in for other services.

I know Travis is (will be) a legacy thing, but perhaps we should just fill in the variables for the service we recommend by default (github), and instruct the user on what the variables would be for Travis or Circle CI or something else.

I think all of this might become somewhat irrelevant when the major refactor happens anyway, so I'm not too concerned.

Reply:
> I assume the `:-` is like an `or`?

If the first variable is unset, return the second. So yes like a Python `or` that returns the actual values rather than True / False.

> One other option, I think, is to just leave the github ones, and make comments about how to fill them in for other services.

I'd like to keep existing Travis CI setups working if we can. The elegance of this `ci/deploy.sh` script is not super important. Users usually do not touch it and hopefully soon, we'll be moving it out of individual manuscript repositories.